src/uthash.h

```diff
@@ -115,6 +138,16 @@ typedef unsigned char uint8_t;
 /* calculate the hash handle from element address elp */
 #define HH_FROM_ELMT(tbl,elp) ((UT_hash_handle *)(((char*)(elp)) + ((tbl)->hho)))
 
+#define HASH_ROLLBACK_BKT(hh, head, itemptrhh)                                   \
+do {                                                                             \
+  struct UT_hash_handle *_hd_hh_item = (itemptrhh);                              \
+  unsigned _hd_bkt;                                                              \
+  HASH_TO_BKT(_hd_hh_item->hashv, (head)->hh.tbl->num_buckets, _hd_bkt);         \
+  (head)->hh.tbl->buckets[_hd_bkt].count++;                                      \
```

Comment:
@Quuxplusone `count++` is correct. `HASH_DELETE_HH` decreases the count, but if bucket resize fails, the count was **not** increased. So it needs to be increased when rolling back to remain unchanged.